x/mongo/driver/topology/rtt_monitor.go

```diff
@@ -89,6 +91,8 @@ func (r *rttMonitor) start() {
 
 	var conn *connection
 	defer func() {
+		r.started = false
```

Comment:
The `rttMonitor`, as currently implemented, is not safe to stop and then restart (a new `rttMonitor` instance must be created). Setting `started` to false might imply to another dev that it can be re-started, which is not accurate. We should remove this line and only set it to `true` in `connect`.